src/gar.lua

```diff
@@ -339,7 +349,7 @@ function gar.delegateStake(from, target, qty, currentTimestamp)
 		error("Cannot delegate to your own gateway, use increaseOperatorStake instead.")
 	end
 
-	if balances.getBalance(from) < qty then
+	if not balances.walletHasSufficientBalance(from, qty) then
```

Comment:
Could even save a few lines around the code with a `balances.assertSufficientBalance(from, qty)`